### PR TITLE
Swagger docs

### PR DESCRIPTION
**Pull request by kostmetallist**
Screenshot from 2020-06-23 19-18-34

This PR brings us an opportunity for getting an API documentation in an easy way. Now, backend is configured to expose `localhost:5000/docs` for Swagger UI, where you can find all the set endpoints along with descriptions and parameter types. There was a slight reorganization of API structure, so for the moment root resources are disabled, and all the valid endpoints have their base path either as `/resources` or `/meta`.

@xelav FYI this is likely to blow up your connection to the API so please make sure React configuration is properly adjusted.

### CODE REVIEW

File: rearguarde/api/meta.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+from flask import current_app
+from flask_restx import Api, Resource
+
+
+about_string = """
```
Comment (bozzyk) [inline rearguarde/api/meta.py:5]: As I understand, this is a constant. If yes - I think we have to name constants in upper-case manner. If this is the variable in common sense then why is it used in global namespace?

Comment (kostmetallist) [reply rearguarde/api/meta.py:5]: It has been spontaneously added at the very beginning of `/about` lifetime. Agree, it is inaccurate now. I think, there is actually no point in holding such a resource. I'm going to remove it now.

File: rearguarde/api/resources.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        },
+        'year_founded': {
+            'description': 'Artist\'s initiation date (only year is taken into account)',
+            'type': 'string',
+            'format': 'date',
+            'pattern': '^\d{4}-\d{2}-\d{2}$',
+            'paramType': 'query',
+            'maxLength': 10,
+        },
+    }
```
Comment (xelav) [inline rearguarde/api/resources.py:40]: I feel a bit iffy about this. On one hand, these params takes most of the files and makes it less readable, On other hand, this placing makes the most sense compared to the separate file option.
I think the current file is fine as it is. But if there is some code to be added, these params variables should be moved somewhere else. I leave this resolved as a single notion.

Comment (artem-bondar) [inline rearguarde/api/resources.py:40]: Please, add spaces here, they're free...

Comment (kostmetallist) [reply rearguarde/api/resources.py:40]: I got your idea here. Later on, it won't be a problem to move these parameters into a separate location.

Comment (kostmetallist) [reply rearguarde/api/resources.py:40]: Will do

**Pull request merged**